src/ostorlab/agent/message/proto/v3/asset/domain_name/domain_name.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto2";
```

Comment:
domaine_name.proto in a dir called domaine_name, not directly in asset/ correct ?